Deploying micro integrator with helm (#842)
wso2/docs-mi: en/docs/install-and-setup/setup/deployment/deploying-micro-integrator-with-helm.md

# Deploy the Micro Integrator on Kubernetes using Helm resources

> Follow the instructions given below to deploy the Micro Integrator on Kubernetes (K8s) using Helm resources.

Reviewer: ```suggestion
Follow the instructions below to deploy the Micro Integrator on Kubernetes (K8s) using Helm resources.
```

    
- Set up a [Kubernetes cluster](https://kubernetes.io/docs/setup/#learning-environment).
    
- Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). 

    !!! Note
        Helm resources for WSO2 product deployment patterns are compatible with the ingress-nginx [`controller-v1.11.2`](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2) release.

## Step 1 - Get the Helm resources

Check out the Helm Resources for the WSO2 Micro Integrator Git repository.

1. Open a terminal and navigate to the location where you want to save the local copy.
2. Clone the Micro Integrator Git repository with Helm resources:

    ```bash
    git clone https://github.com/wso2/helm-mi.git
    ```

This creates a local copy of [`wso2/helm-mi`](https://github.com/wso2/helm-mi/), which includes all the Helm Resources for WSO2 Micro Integrator.

Let's refer to the root folder of the local copy as `<HELM_HOME>`.

## Step 2 - Update the deployment configurations 

> Follow the steps given below to configure how your Micro Integrator deployment should be set up.

Reviewer: ```suggestion
Follow the steps below to configure how your Micro Integrator deployment should be set up.
```

1. Open the `values.yaml` file in the `<HELM_HOME>/mi` folder of your local copy.

2. Use the following guidelines to update the deployment configurations:

    - **Update the WSO2 subscription details**
    

>         You can update the user name and password in the following section. If you don't have an active WSO2 subscription, leave these parameters empty.

Reviewer: ```suggestion
        You can update the username and password in the following section. If you don't have an active WSO2 subscription, leave these parameters empty.
```

2.  Clone the Micro Integrator Git repository with Helm resources:

    ```bash
    git clone https://github.com/wso2/helm-mi.git
    ```

This creates a local copy of [`wso2/helm-mi`](https://github.com/wso2/helm-mi/), which includes all the Helm Resources for WSO2 Micro Integrator.

Let's refer to the root folder of the local copy as `<HELM_HOME>`.

## Step 2 - Update the deployment configurations 

Follow the steps given below to configure how your Micro Integrator deployment should be set up.

> 1.  Open the `values.yaml` file in the `<HELM_HOME>/mi` folder of your local copy.

Reviewer 2: Why adding two spaces after `1.`. Check and fix all the pages in this file.
